hack/install.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -e
+
+: ${KUBE_NAMESPACE:=m4d-system}
+: ${PORT_TO_FORWARD:=8200}
+
+source third_party/vault/vault-util.sh
+
+kubectl create ns $KUBE_NAMESPACE || true
+
+kubectl apply -f manager/config/prod/deployment_configmap.yaml -n $KUBE_NAMESPACE
+
+make cluster-prepare
+
+# Install third party components
+make -C third_party/vault deploy
+make -C third_party/egeria deploy
+make -C third_party/opa deploy
+
```

Comment:
@tomersolomon1 can we add a flag to enable the deployment of these components? this would allow easier deployment of different connectors. And have the option to override the flags with settings that we define before running the script?

Reply:
I can add a flags like `WITHOUT_X`, so this code snippet would become:

Reply:
@tomersolomon1 any way to make it a positive statement? i.e., WITHOUT ==> With ?
if so, I am good.

Reply:
Currently in the code-base the convention is to use WITHOUT_X. We could use here WITH_X, but then the script won't be consistent with the rest of the script.

Reply:
+1 for the suggestion by @tomersolomon1

Reply:
I would like it changed, the positive way is much better. please handle through an independent issue.